src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
```

Comment:
As mentors said variable better named by full word: prev -> previous;

Reply:
Yes, thank you, I missed it)